src/App.tsx

```diff
@@ -1,7 +1,37 @@
-import React from 'react';
+import { Component } from 'react';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <p className="App__message">The last pressed key is [Enter]</p>
-  </div>
-);
+type State = {
+  click: string;
+};
+
+export class App extends Component<{}, State> {
+  state: Readonly<State> = {
+    click: '',
```

Comment:
better rename to `pressedKey` or `pressedButton`, to make the variable more meaningful.